src/gp_engine.hpp

```diff
@@ -92,7 +105,7 @@ class GPEngine {
   // These parameters are indexed in the same way as sbn_parameters_ in
   // gp_instance.
   EigenVectorXd branch_lengths_;
-  EigenVectorXd log_likelihoods_;
+  EigenMatrixXd log_likelihoods_;
```

Comment:
I don't think that `log_likelihoods_` follows 

    // These parameters are indexed in the same way as sbn_parameters_ in
    // gp_instance.

Please move below and explain what the rows and columns of this matrix are in a comment.

Reply:
Done.